controllers/errors.controllers.js

```diff
@@ -0,0 +1,4 @@
+exports.handleRouteNotFoundError = (req, res, next) => {
+    return res.status(404).send({msg: 'Not found'})
+    .catch(next);
```

Comment:
No need for a `.catch` block here, or to return the res.status
This is the end of the line for our promise chain and we just need to send the response ✉️

Reply:
Deleted the catch block.